src/controllers/settings_controller.js

```diff
@@ -228,24 +229,108 @@ SettingsController.post('/skiptunnel', async (request, response) => {
   }
 });
 
+
 SettingsController.get('/tunnelinfo', async (request, response) => {
   try {
+    // Check to see if we have a tunnel endpoint first
     const result = await Settings.get('tunneltoken');
+    let localDomain;
+    let mDNSstate;
+    let tunnelEndpoint;
+
     if (typeof result === 'object') {
-      const endpoint = `https://${result.name}.${
-        config.get('ssltunnel.domain')}`;
-      response.send(endpoint);
-      response.status(200).end();
+      console.log(`Tunnel domain found. Tunnel name is: ${result.name} and 
```

Comment:
nit: If you add line breaks inside a template string (i.e. with backticks), the line breaks and subsequent space will be inside the resulting string as well. As such, this will actually log 4 lines.

Reply:
done. used '\' this method.